### ISSUE TITLE
Full support for iMessage extension

### ISSUE THREAD

**Issue by jvence** [HernanZh/CordovaStickers#1]
Thank you for working on this project. Will you providing full integration support between an iMessage extension and a cordova app beyond stickers.

**Comment by HernanZh** [comment]
I'd like to, but editing XCode projects is ridiculously difficult on its own already and I hardly got this plugin to work. Also an iMessage extension could be anything, so a basic plugin probably wouldn't do more than add the extension target and add some source code. Who knows what else that extension need, that would depend on the app. I might try something in the future, but not anytime soon.

**Comment by charlesbodman** [comment]
Yeah, it's sad that apple hasn't made this easy for anyone. The lack of documentation is ridiculous
